gitlab-sso.md

```diff
@@ -0,0 +1,19 @@
+# Configuring GitLab Single-Sign-On
+
+The following steps can be used to configure Mattermost to use GitLab as a single-sign-on (SSO) service.
+
+1. Login to your GitLab account and go to the Applications section either in Profile Settings or Admin Area.
+2. Add a new application called "Mattermost" with the following as Redirect URIs:
+  * `<your-mattermost-url>/login/gitlab/complete` (example: http://localhost:8065/login/gitlab/complete)
+  * `<your-mattermost-url>/signup/gitlab/complete`
+
+3. Submit the application and copy the given _Id_ and _Secret_ into the appropriate _SSOSettings_ fields in config/config.json
+
+4. Also in config/config.json, set _Allow_ to `true` for the _gitlab_ section, leave _Scope_ blank and use the following for the endpoints:
```

Comment:
@it33 can you tell me something about `Scope`? If it is not blank what value can it be and what for can it be used?